www/src/pages/table/NodeSearchComponent.tsx

```diff
@@ -4,10 +4,12 @@ import { Select, Spin } from "antd";
 import { observer } from "mobx-react";
 import React, { useMemo, useState } from "react";
 import { SequentialFuncInvoker } from "../../misc";
-import { ClassTextSearchResult } from "../../models/ontology/ClassStore";
```

Comment:
rename it to ClassTextSearchResult, as we are using same naming convention for Entity and properties.